src/content/advanced/gitops/bases/index.md

```diff
@@ -0,0 +1,155 @@
+---
+linkTitle: Creating a base template clusters
+title: Creating a base template for workload clusters
+description: How to create a base template to create workload clusters with different configurations.
+weight: 70
+menu:
+  main:
+    parent: advanced-gitops
+    identifier: advanced-gitops-bases
+user_questions:
+  - How can I create an base template for workload clusters in GitOps?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-honeybadger
+last_review_date: 2023-01-20
+---
+
+# Add a CAPx (CAPI) Workload Cluster template (cluster App based)
+
+Our CAPx (CAPI provider-specific clusters) are delivered by Giant Swarm as a set of two applications. The first one is an [App Custom Resource](https://docs.giantswarm.io/platform-overview/app-platform/)(CR) with a Cluster instance definition, while the second one is an App CR containing all the default applications needed for a cluster to run correctly. As such, creating a CAPx cluster means that you need to deliver two configured App CRs to the Management Cluster.
+
+Adding definitions can be done on two levels: shared cluster template and version-specific template, see [create shared template base](#create-shared-template-base) and [create versioned base](#create-versioned-base-optional).
+
+**IMPORTANT**, CAPx configuration utilizes the [App Platform Configuration Levels](/getting-started/app-platform/app-configuration/#levels), in the following manner:
+
+- cluster templates provide default configuration via App' `config` field,
+- cluster instances provide custom configuration via App' `extraConfig` field, which is overlaid on top of `config`.
```

Comment:
This - what is laid on top of what other layers - actually depends on the `priority` field set to `extraConfig` entries. But his may too much of a detail here, the linked docs above describes it with examples and all.